src/rust/cryptography-x509-validation/src/policy/mod.rs

```diff
@@ -0,0 +1,126 @@
+// This file is dual licensed under the terms of the Apache License, Version
+// 2.0, and the BSD License. See the LICENSE file in the root of this repository
+// for complete details.
+
+use std::collections::HashSet;
+
+use once_cell::sync::Lazy;
+
+use cryptography_x509::common::{
+    AlgorithmIdentifier, AlgorithmParameters, RsaPssParameters, PSS_SHA256_HASH_ALG,
+    PSS_SHA256_MASK_GEN_ALG, PSS_SHA384_HASH_ALG, PSS_SHA384_MASK_GEN_ALG, PSS_SHA512_HASH_ALG,
+    PSS_SHA512_MASK_GEN_ALG,
+};
+
+pub static WEBPKI_PERMITTED_ALGORITHMS: Lazy<HashSet<AlgorithmIdentifier<'_>>> = Lazy::new(|| {
```

Comment:
@facutuesca Might be worth adding a comment linking to the CA/B Forum BRs doc here (with a reference to the pages these are from)

(Sorry, I forgot to do this myself 😅)

Reply:
Done!